server/log_rpc_server_test.go

```diff
@@ -539,12 +539,12 @@ func TestGetLeavesByLeafValueHashInvalidHash(t *testing.T) {
 
 	// Should have succeeded at RPC level
 	if err != nil {
-		t.Fatalf("GetLeavesByLeafValueHash() = %v", err)
+		t.Fatalf("GetLeavesByLeafValueHash(hash empty) = %v, want nil", err)
```

Comment:
Not sure what you mean. In this case we want no RPC level error but an app status error. In the case below we expect a TX specific error but the message was wrong after splitting the cases. I've tried again but I don't know if I've addressed the point.